doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -66,9 +66,13 @@ lose their role of parameters. In the same vein, every symbol of at
 least 3 characters and starting with a simple quote must be quoted
 (then it starts with two single quotes). Here is an example.
 
+.. coqtop:: none
+
+   Reserved Notation "'IF' c1 'then' c2 'else' c3" (at level 200, right associativity).
```

Comment:
I suggest to just add the explicit `(at level 200, right associativity)` to the notation below as this will avoid puzzling issues for users who try to replicate the example locally.

Reply:
Done, this should be ready now.